airbyte-cdk/java/airbyte-cdk/db-sources/src/test/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIteratorTest.kt

```diff
@@ -34,6 +34,7 @@ class DebeziumRecordIteratorTest {
                 { false },
                 mock(),
                 Duration.ZERO,
+                false
```

Comment:
It would be good to add a comment here to indicate we are not turning off a heartbeat.